guides/advanced_guides/faceted_search.md

```diff
@@ -94,40 +92,208 @@ By introducing facets to MeiliSearch, new search query parameters were also adde
 
 ### The facet filters
 
-You can filter on facets to narrow down your results based on criteria with the `facetFilters` attribute.
-Learn more about `facetFilters` in **[the search parameters](/guides/advanced_guides/search_parameters.md#facet-filters)**.
-
-#### Example
-
-Given a clothing dataset, suppose you want to retrieve all t-shirts with a nautical pattern. You would then use:
+Because facets relate specifically to a set of documents, they give an overview of the records and help to understand what kind of information can be searched. Facets then help users navigate through their search results.
 
-```bash
-$ curl --get 'http://localhost:7700/indexes/clothing/search' \
-    --data-urlencode 'q=nautical' \
-    --data-urlencode 'facetFilters=["kind:t-shirt"]'
-```
+They can filter on facets to narrow down their results based on criteria with the `facetFilters` attribute.
+Learn more about `facetFilters` in **[the search parameters](/guides/advanced_guides/search_parameters.md#facet-filters)**.
 
 ### The facets distribution
 
 Since the users can have a visual clue about the range of categories available in the UI, they can easily know how many search results are found for each category.
 
 In the example below, on the [snowleader.co.uk website](https://www.snowleader.co.uk), the number in parentheses matches how many search results each facet relates to.
 
-![snowleader boot](/faceted-search/facets-dist-boot.png)
-> When searching for "Boot", 791 outdoor boots have been found. If the user wants to buy outdoor boots, they can narrow down their search results by clicking the Outdoor category. Image from [snowleader.co.uk](https://www.snowleader.co.uk).
-
 ![snowleader ski pants](/faceted-search/facets-dist-pants.png)
-> If a user is looking for ski pants available in XXL size, they can select the right size. Thus they won't waste their time looking at items they are not interested in buying. Image from [snowleader.co.uk](https://www.snowleader.co.uk).
+> If a user is looking for ski pants available in XXL size, they can select the right size. Thus they won't waste their time looking at items they are not interested in buying.
```

Comment:
It would probably be better to find another example more related to movies :)

Reply:
Yes, you're right! I updated it to align all examples.